optlang/tests/test_cplex_interface.py

```diff
@@ -497,6 +499,98 @@ def test_qp_non_convex(self):
             model.optimize()
             self.assertAlmostEqual(model.objective.value, 2441.999999971)
 
+    class InfeasibleMIPTestCase(unittest.TestCase):
```

Comment:
I'd rather keep them separate since the intention of the tests is something different.